docs/Contributing/guides/README.md

```diff
@@ -0,0 +1,35 @@
+# Fleet development guides
+
+This directory contains guides for common development tasks in Fleet.
+
+## General guides
+
+- [Seeding Data](seeding-data.md) - Guide for seeding data in Fleet
+- [Committing Changes](committing-changes.md) - Guide for committing changes to Fleet
+- [Fleetctl Apply](cli/fleetctl-apply.md) - Guide for using fleetctl apply
+- [Simulate Slow Network](simulate-slow-network.md) - Guide for simulating a slow network
+- [Troubleshooting Live Queries](troubleshooting-live-queries.md) - Guide for troubleshooting live queries
+- [Enroll hosts with plain osquery](enroll-hosts-with-plain-osquery.md) - Guide for enrolling hosts with plain osquery
+
+## UI development
+
+- [Fleet UI Testing](ui/fleet-ui-testing.md) - Guide for testing the Fleet UI
+- [Generating UI Component Boilerplate](ui/generating-ui-component-boilerplate.md) - Guide for generating UI component boilerplate
+- [Design QA Considerations](ui/design-qa-considerations.md) - Guide for design QA considerations
+
+## API development
```

Comment:
If we want a readme at the root of API development, we should just link to it here (same for UI, same for integrations) rather than having separate lists both here and in the API root, to avoid drift.

Reply:
The premise with this change is that every directory should have its own README. I don't think it's too onerous for us to remember to update both, but I'm open to changing this later.